plugin/pkg/scheduler/schedulercache/node_info_test.go

```diff
@@ -0,0 +1,253 @@
+package schedulercache
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/kubernetes/pkg/api/v1"
+	"reflect"
+	"testing"
+)
+
+var (

```

Comment:
Not a big fan of having a global across tests vs. an init f(n) that passes back an object that is used in the tests.